xcube/cli/gen.py

```diff
@@ -68,8 +68,10 @@
 @click.option('--append', '-a', is_flag=True,
               help='Append successive outputs.')
```

Comment:
Now, we should really think about dropping `--append`. Append-mode should be default, that is, if a cube exists append new slices, if not create one.

It would now be better to introduce a new option `--new` to force creating new, non-existent data cubes. If `--new` is given and the cube exists, the command then fails.